 color_example:
  upper_orange: '0,0,0'

  lower_orange: '179,255,255'

  upper_red: '15,254,255'

  lower_red: '0,182,116'

  upper_yellow: '0,0,0'

  lower_yellow: '179,255,255'

  upper_green: '88,255,255'

  lower_green: '49,67,151'

  upper_black: '0,0,123'

  lower_black: '0,0,29'
